pkg/clients/edgeconnect/client.go

```diff
@@ -276,3 +280,37 @@ func (c *client) CreateEdgeConnect(name string, hostPatterns []string, oauthClie
 
 	return response, nil
 }
+
+// GetEdgeConnects returns list of edge connects
+func (c *client) GetEdgeConnects(name string) (GetEdgeConnectsResponse, error) {
```

Comment:
```suggestion
func (c *client) List(name string) (ListResponse, error) {
```
if we want to follow Andrii's pattern, which I also would prefer